tests/python/relay/aot/test_cpp_aot.py

```diff
@@ -145,6 +145,26 @@ def test_mobilenet(enable_usmp, target_kind):
     assert (runner.get_output(0).asnumpy() == list(ref_outputs.values())[0]).all()
 
 
+def test_module_list():
+    x = tvm.relay.var("x", tvm.relay.TensorType([1], dtype="float32"))
+    expr = tvm.relay.add(x, tvm.relay.Constant(tvm.nd.array(np.array([1], dtype="float32"))))
+    mod = tvm.relay.build(
```

Comment:
nit: could pass mod_name = 'foo' here and then test that to verify the knob works

Reply:
Done.